configs/llm-router/aws.yaml

```diff
@@ -0,0 +1,19 @@
+head_node_type:
+  name: head
+  instance_type: p4de.24xlarge
```

Comment:
A100 head nodes are not available in Hosted OA - if this template will be exposed in Hosted OA, can we use the serverless config (just directly request a GPU resource of A100-80G and allow the autoscaler to upscale it?)

If we are not planning on exposing this through OA, then it doesn't matter as much. But it's still better practice to run workloads on workers and use cheap CPU nodes for development.

Reply:
would you consider A10 as cheap gpu? I have enabled training on A10s and launched jobs successfully with it, so I can update this config